components/sidebar_toggle.py

```diff
@@ -50,7 +50,17 @@ def add_callback(app):
         def _toggle_sidebar(n_clicks):
             SidebarToggle.is_open = not SidebarToggle.is_open
             return (
-                {"display": "block"} if SidebarToggle.is_open else {"display": "none"},
+                {
```

Comment:
This is fixed in f18392d223cf67d3e00cdae9d44df1f4ee3edfdb. So merging sidebar-toggler-refactor into this branch should be enough.